src/Representation/KeyMetadata.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Fschmtt\Keycloak\Representation;
+
+/**
+ * @method string|null getProviderId()
+ * @method int|null getProviderPriority()
+ * @method string|null getKid()
+ * @method string|null getStatus()
+ * @method string|null getType()
+ * @method string|null getAlgorithm()
+ * @method string|null getPublicKey()
+ * @method string|null getCertificate()
+ * @method string|null getUse()
+ * @method self withProviderId()
```

Comment:
The arguments are missing for the `::with` methods

```suggestion
 * @method self withProviderId(?string $providerId)
```